save-common/src/commonMain/kotlin/org/cqfn/save/core/utils/ProcessParseSarifException.kt

```diff
@@ -0,0 +1,6 @@
+package org.cqfn.save.core.utils
+
+/**
+ * An [Exception] that can be thrown in case of failed to parse sarif
+ */
+open class ProcessParseSarifException(message: String) : Exception(message)
```

Comment:
```suggestion
open class SarifParsingException(message: String) : Exception(message)
```
Also, why `open class`?

Reply:
Fixed